add(deploy)(v7.9): add deploy to hana  parameter description (#758)
cap-js/docs: guides/databases-hana.md

If you run into issues, see the [Troubleshooting](../get-started/troubleshooting#hana) guide.

#### Deploy Parameters

> When using `--to hana` to deploy your app to SAP HANA database you can specify the service name and logon information in several ways.

Reviewer: App? Previously we say "_just the database parts..._". How about omitting it here?
```suggestion
When using the option `--to hana`, you can specify the service name and logon information in several ways.
```

<br>

`cds deploy --to hana`

> In this case the service name either comes from the environment variable `VCAP_SERVICES` or is defaulted from the project name, for example, `myproject-db` with `myproject-db-key`. Service instances and key either exist and will be used, or otherwise they're created.

Reviewer: ```suggestion
In this case the service name and service key either comes from the environment variable `VCAP_SERVICES` or is defaulted from the project name, for example, `myproject-db` with `myproject-db-key`. Service instances and key either exist and will be used, or otherwise they're created.
```
